library/src/main/java/com/bartoszlipinski/viewpropertyobjectanimator/ViewPropertyObjectAnimator.java

```diff
@@ -22,8 +22,8 @@
 import android.animation.ValueAnimator;
 import android.annotation.SuppressLint;
 import android.os.Build;
-import android.support.v4.util.ArrayMap;
-import android.support.v4.view.ViewCompat;
+import androidx.collection.ArrayMap;
```

Comment:
Let's remove the whole `ArrayMap` (migrate to `HashMap`). 
The performance benefits with such small dataset is probably negligible and removing it will let us remove the `collection` dependency completely.

Reply:
If we remove/replace the single method we take in from ViewCompat, we can drop support library in it's entirety (if we drop percent layout). What do you think?

Reply:
Removing `ViewCompat.isAttachedToWindow` is a good idea. Let's copy the internals of the method to the library.
As far as the percent library goes, let's keep it, especially since we use `compileOnly` with it.